### PR TITLE
multi-dc: update meeting notes

### PR DESCRIPTION
**Pull request by hicqu**
Signed-off-by: qupeng <qupeng@pingcap.com>

### CODE REVIEW

File: wg/multiple-dc-enhancemant/README.md [hunk 1]
```diff
@@ -8,6 +8,10 @@ TiDB across multiple data centers.
 List the stakeholder SIGs of this Working Group.
 
 ## Meetings
+* 2019-12-03
+  > Send an issue to etcd to show proto changes.
+  > Progresses of delegated peers are maintained on delegates.
```
Comment (dcalvin) [inline wg/multiple-dc-enhancemant/README.md]: what do you mean by "are maintained on delegates"?

Comment (hicqu) [reply wg/multiple-dc-enhancemant/README.md]: Something like "progresses are managed on delegates". Please help me to improve it.

**Comment by tisonkun** [comment]
closed as invalid any more.

**Pull request closed without merging**